modules/luxon/src/impl/english.js

```diff
@@ -0,0 +1,233 @@
+import * as Formats from './formats.js';
+import { pick } from './util.js';
+
+function stringify(obj) {
+  return JSON.stringify(obj, Object.keys(obj).sort());
+}
+
+/**
+ * @private
+ */
```

Comment:
- [ ] Great job for the displayDate() function implementation, :clap: but I think it will be better if you can use arrow syntax for functions instead of a  function declaration for more information [JavaScript Function Expressions](https://www.w3docs.com/learn-javascript/function-expressions.html)